<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Captain Shutdown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Hours</string>
         </property>
         <property name="buddy">
          <cstring>hLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusLineEdit" name="hLineEdit">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Minutes</string>
         </property>
         <property name="buddy">
          <cstring>mLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusLineEdit" name="mLineEdit">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Seconds</string>
         </property>
         <property name="buddy">
          <cstring>sLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusLineEdit" name="sLineEdit">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="abButton">
         <property name="text">
          <string>&amp;Abort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>&amp;Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Hours</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdHours">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Minutes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdMinutes"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Seconds</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdSeconds"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCaptain_Shutdown">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="resetAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="aboutCptAction"/>
     <addaction name="aboutQtAction"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <widget class="QMenu" name="menuPassword">
    <property name="title">
     <string>Password</string>
    </property>
    <addaction name="setPAction"/>
    <addaction name="editPAction"/>
    <addaction name="removePAction"/>
   </widget>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>Note</string>
    </property>
    <addaction name="addNoteAction"/>
   </widget>
   <addaction name="menuCaptain_Shutdown"/>
   <addaction name="menuPassword"/>
   <addaction name="menuNote"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="resetAction">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="statusTip">
    <string>Reset all values to 0</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Close the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_QtWidgets">
   <property name="text">
    <string>About QtWidgets</string>
   </property>
   <property name="statusTip">
    <string>Some info about QtWidgets</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="aboutCptAction">
   <property name="text">
    <string>About CptShutdown</string>
   </property>
   <property name="statusTip">
    <string>Some info about Cpt Shutdown</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About QtWidgets</string>
   </property>
   <property name="statusTip">
    <string>Some info about QtWidgets</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="setPAction">
   <property name="text">
    <string>Set Password</string>
   </property>
   <property name="statusTip">
    <string>Set a new password</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="editPAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Password</string>
   </property>
   <property name="statusTip">
    <string>Edit your current password</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="removePAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Password</string>
   </property>
   <property name="statusTip">
    <string>Remove the current password</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="addNoteAction">
   <property name="text">
    <string>Add/Edit/Remove Note</string>
   </property>
   <property name="statusTip">
    <string>Add, edit or remove notes</string>
   </property>
  </action>
  <action name="editNoteAction">
   <property name="text">
    <string>Edit/Remove Note</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FocusLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">focuslineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hLineEdit</tabstop>
  <tabstop>mLineEdit</tabstop>
  <tabstop>sLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>abButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lineChanged()</slot>
 </slots>
</ui>
